tf2_ros/CMakeLists.txt

```diff
@@ -105,6 +119,7 @@ install(TARGETS
 install(TARGETS
   buffer_server
   static_transform_publisher
+  static_transform_component
```

Comment:
@clalancette errr seems `static_transform_publisher` is taken... What would you prefer in its place?

Reply:
Oh, haha.  I didn't even realize.

Looking at this again, what this ends up being is a node wrapping the `StaticTransformBroadcaster` class.  So how about naming the class `StaticTransformBroadcasterNode`, and the file `static_transform_broadcaster_node.cpp`?